source/development_process/configuring_git.rst

```diff
@@ -0,0 +1,71 @@
+Configuring Git
```

Comment:
Good idea, I'll probably pull this whole thing out of the PR. I wrote half of it by accident.